python/uv/private/current_toolchain.bzl

```diff
@@ -30,7 +30,9 @@ def _current_toolchain_impl(ctx):
     # Bazel requires executable rules to create the executable themselves,
     # so we create a symlink in this rule so that it appears this rule created its executable.
     original_uv_executable = toolchain_info.uv_toolchain_info.uv[DefaultInfo].files_to_run.executable
-    symlink_uv_executable = ctx.actions.declare_file("uv_symlink_{}".format(original_uv_executable.basename))
+
+    # Use `uv` as the name of the binary to make the help message well formatted
```

Comment:
nit: The only reason I went with a weird name, was because I was worried about Windows. I might be wrong, but I thought Windows requires(ed) that executables have the `.exe` suffix?